tests/phpunit/AgenticCommerce/WC_Stripe_Agentic_Commerce_Tax_Calculator_Test.php

```
@@ -0,0 +1,426 @@
+	 */
+	public function test_calculate_returns_empty_rates_when_tax_disabled() {
+		update_option( 'woocommerce_calc_taxes', 'no' );
+
+		$event      = $this->build_event( [ $this->product ] );
+		$line_items = $this->calculator->extract_line_items_from_customization_hook( $event );
+		$result     = $this->calculator->calculate( $event, $line_items );
+
+		$this->assertArrayHasKey( 'line_items', $result );
+		$this->assertCount( 1, $result['line_items'] );
+		$this->assertEmpty( $result['line_items'][0]['tax_rates'] );
+	}
```

PHPMD (2.15.0)

[error] The method test_calculate_returns_empty_rates_when_tax_disabled is not named in camelCase. (undefined)

(CamelCaseMethodName)